libmdnsd/CMakeLists.txt

```diff
@@ -63,13 +67,33 @@ if(MDNSD_COMPILE_AS_CXX)
 endif()
 
 if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES})
+    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES})
+    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 endif()
 
 if(WIN32 OR MINGW)
   target_link_libraries(libmdnsd wsock32 ws2_32)
 endif()
 
 target_compile_definitions(libmdnsd PRIVATE -DMDNSD_DYNAMIC_LINKING_EXPORT)
+
+include(GNUInstallDirs)
+
+install(
+    TARGETS ${PROJECT_NAME}
+    EXPORT ${PROJECT_NAME}Targets
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
```suggestion
    LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
    ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
    RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
```

Reply:
Destinations do not need to be strings explicitly, they can take path variables as well

Reply:
The examples at https://cmake.org/cmake/help/latest/command/install.html
don't use quotes indeed.
I don't see an example of using a path variable.

Reply:
Under `CONFIGURATIONS` example an explicit Path variable is used, here is the snippet for it:

```cmake
install(TARGETS target
        CONFIGURATIONS Release
        RUNTIME DESTINATION Release/bin)
```

`Release/bin` is a path variable, if it were `"Release/bin"` it would be a string. Both work just fine since CMake 3.6 at least from what I used.

Reply:
`Release/bin` is also a string? And paths in cmake are strings.
By enclosing them between `"`, you allow spaces in paths.